pyaerocom/aeroval/config/emep/reporting_base.py

```diff
@@ -162,7 +162,13 @@ def clean_filters(cfg: dict, obs_pattern: str) -> dict:
     return CFG
 
 
-def get_CFG(reportyear, year, model_dir, omit_stations_path=DEFAULT_OMIT_STATION_PATH) -> dict:
+def get_CFG(
+    reportyear,
+    year,
+    model_dir,
+    omit_stations_path=DEFAULT_OMIT_STATION_PATH,
+    remove_urban_stations=True,
```

Comment:
Can we call it `remove_urban_ebas_stations`? We have also the EEA stations with their own split of urban/background.